index.html

```diff
@@ -53,8 +53,23 @@ <h2 class="connect-txt">LET'S CONNECT</h2>
                     <li class="icon"><img src="./images/medium.png" alt="Medium logo"></li>
                 </ul>
             </div>
-        </div>
-    </header>
+        </div> 
+    </header> 
+    <!-- menu section -->
+    <section class="menu">
+        <i class="fa-solid fa-xmark"></i>
+        <ul class="menu-item">
+            <li>
+                <a href="#projects-container" class="meu-item">Portfolio</a>
+            </li>
+            <li>
+                <a href="#about" class="meu-item">About</a>
+            </li>
+            <li>
+                <a href="#contact" class="meu-item">Contact</a>
+            </li>
+        </ul>
+    </section>  
```

Comment:
- [ **OPTIONAL** ] For good semantics tags, I suggest you wrap all navigation content in a `<nav>` tag and not a `<section>` tag